### ISSUE TITLE
My profile is not visible and my games are waiting to be reviewed by moderator its been more than year

### ISSUE THREAD

**Issue by ZahidServers** [itchio/itch.io#1190]
Some years ago I started my itch.io account to distribute my games but my profile is still not visible from other browsers and my games are waiting to be reviewed by moderators meanwhile I had switched to gamejolt. I reporting this because I was planning to launch my new game Fighting Evil Regime which I made with gameguru engine also itch.io along with gamejolt. My games on itch.io were previously visible although my profile was not. But now even games are waiting to be reviewed by a moderator. I am not sure about how games are reviewed here but hope the problem is soon fixed.
Here is my profile https://zahidgames.itch.io/
Thanks in advance!

**Comment by leafo** [comment]
It looks like your account has been flagged for spamming. The post on your account that was caused it has since been deleted (it's been about 2 years, as you mentioned), so I don't have further details about the specific cause.

**Comment by ZahidServers** [comment]
> you have links to some hosting service

Yeah that's my hosting company basically my games are for free so I ask people to support me by using my other services such as animation and web hosting(On itch.io only hosting) but as far as I remember I never posted it on any other peoples pages. I am not aware that if its considered as spamming on your platform to promote commercial links on own Page. 
Thanks for your reply.
I have removed the links also some games due large size I gave my website link now I have removed it and uploaded the installer of game itself on itch.io. I hope that my account and games will be public again as i removed those links from my page. 
Thanks in advance.
